README.md

```diff
@@ -11,7 +11,7 @@
 
 ## Overview
 
-Raito is a zero-knowledge Bitcoin client implemented in Cairo. It aims to provide trustless validation of the Bitcoin blockchain through STARK proof verification. It is heavily inspired by [ZeroSync](https://github.com/ZeroSync/ZeroSync) project.
+Shinigami-consensus is a zero-knowledge Bitcoin client implemented in Cairo. It aims to provide trustless validation of the Bitcoin blockchain through STARK proof verification. It is heavily inspired by [ZeroSync](https://github.com/ZeroSync/ZeroSync) project.
```

Comment:
Maybe let's just keep Shinigami? 
This also rises a question whether we should rename repos (probably it's time)